app/src/main/res/layout/list_bp_history_item.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="wrap_content">
+
+  <LinearLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginStart="@dimen/spacing_12"
+    android:layout_marginTop="@dimen/spacing_8"
+    android:layout_marginEnd="@dimen/spacing_12"
+    android:layout_marginBottom="@dimen/spacing_8"
+    android:gravity="center_vertical"
+    android:orientation="horizontal"
+    app:layout_constraintBottom_toTopOf="@id/divider"
+    app:layout_constraintTop_toTopOf="parent">
+
+    <ImageView
+      android:id="@+id/heartImageView"
+      android:layout_width="@dimen/spacing_16"
+      android:layout_height="@dimen/spacing_16"
+      android:layout_gravity="center_vertical"
+      tools:ignore="ContentDescription"
+      tools:src="@drawable/bp_reading_normal" />
+
+    <org.simple.clinic.widgets.TintableCompoundDrawableTextView
+      android:id="@+id/readingsTextView"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_gravity="center_vertical"
+      android:layout_marginStart="@dimen/spacing_12"
+      android:textAppearance="@style/Clinic.V2.TextAppearance.Body1Left.Grey0"
+      tools:text="120 / 80" />
+
+    <TextView
+      style="@style/Clinic.V2.TextAppearance.Button2.Blue1"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="@dimen/spacing_8"
+      android:text="@string/patientsummary_edit" />
+
+    <TextView
+      android:id="@+id/dateTimeTextView"
+      android:layout_width="0dp"
+      android:layout_height="wrap_content"
+      android:layout_weight="1"
+      android:textAlignment="viewEnd"
+      android:textAppearance="@style/Clinic.V2.TextAppearance.Body2Center.Grey1"
+      tools:text="10-Aug-2019" />
+
+  </LinearLayout>
+
+  <View
```

Comment:
I've been steadily getting annoyed with having to manage dividers everywhere manually. I'm wondering if we should investigate using the [`DividerItemDecoration`](https://developer.android.com/reference/androidx/recyclerview/widget/DividerItemDecoration?hl=en) to do this automatically.

Reply:
We can use that, but we may have to extend it to take `marginStart` and `marginEnd`

Reply:
Alright, let's explore this once the feature is complete.